2018-11-14 00:17:05,280 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 00:17:05,283 [open_context_base.py] on_connected:289: Connected : conn_id=1; 
2018-11-14 00:17:05,292 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=1; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 17366925671464366343, 'conn_key': '01F13E60E2B703E9', 'keep_alive_interval': 10};
2018-11-14 00:17:05,292 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 00:17:05,297 [open_context_base.py] on_connected:289: Connected : conn_id=2; 
2018-11-14 00:17:05,300 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=2; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 12899357040136217858, 'conn_key': '06E54060E2B903DB', 'keep_alive_interval': 10};
2018-11-14 00:17:05,312 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 00:17:05,315 [open_context_base.py] on_connected:289: Connected : conn_id=3; 
2018-11-14 00:17:05,318 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=3; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 9728825101490932228, 'conn_key': '034A4560E2BB033E', 'keep_alive_interval': 10};
2018-11-14 02:25:22,846 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:25:22,849 [open_context_base.py] on_connected:289: Connected : conn_id=1; 
2018-11-14 02:25:22,865 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=1; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 2595125613919962114, 'conn_key': '0684370DFFBD030D', 'keep_alive_interval': 10};
2018-11-14 02:25:22,871 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:25:22,876 [open_context_base.py] on_connected:289: Connected : conn_id=2; 
2018-11-14 02:25:22,881 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=2; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 433399991805763589, 'conn_key': '06603D0DFFBF0347', 'keep_alive_interval': 10};
2018-11-14 02:25:22,902 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:25:22,904 [open_context_base.py] on_connected:289: Connected : conn_id=3; 
2018-11-14 02:25:22,908 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=3; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 12899365959390952072, 'conn_key': '827A430DFFC1034D', 'keep_alive_interval': 10};
2018-11-14 02:27:02,835 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:27:02,838 [open_context_base.py] on_connected:289: Connected : conn_id=1; 
2018-11-14 02:27:02,841 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=1; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 4828917627765222793, 'conn_key': '846D916CFFC3038B', 'keep_alive_interval': 10};
2018-11-14 02:27:02,847 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:27:02,851 [open_context_base.py] on_connected:289: Connected : conn_id=2; 
2018-11-14 02:27:02,855 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=2; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 2234846441423328641, 'conn_key': '8355956CFFC5035A', 'keep_alive_interval': 10};
2018-11-14 02:27:02,865 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:27:02,868 [open_context_base.py] on_connected:289: Connected : conn_id=3; 
2018-11-14 02:27:02,871 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=3; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 649581571612393604, 'conn_key': '81C0986CFFC70325', 'keep_alive_interval': 10};
2018-11-14 02:29:44,999 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:29:45,004 [open_context_base.py] on_connected:289: Connected : conn_id=1; 
2018-11-14 02:29:45,009 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=1; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 6702420572904160008, 'conn_key': '04FB380700C903CB', 'keep_alive_interval': 10};
2018-11-14 02:29:45,012 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:29:45,014 [open_context_base.py] on_connected:289: Connected : conn_id=2; 
2018-11-14 02:29:45,023 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=2; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 10809705632089563911, 'conn_key': '09E33C0700CB0345', 'keep_alive_interval': 10};
2018-11-14 02:29:45,039 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 02:29:45,041 [open_context_base.py] on_connected:289: Connected : conn_id=3; 
2018-11-14 02:29:45,044 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=3; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 1370163012144563841, 'conn_key': '8886410700CD0389', 'keep_alive_interval': 10};
2018-11-14 05:15:56,796 [open_context_base.py] on_error:307: Connect error: conn_id=2; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 05:15:56,832 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 05:15:56,927 [open_context_base.py] on_error:307: Connect error: conn_id=1; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 05:15:56,927 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 05:15:56,930 [open_context_base.py] on_error:307: Connect error: conn_id=2; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 05:15:56,944 [open_context_base.py] on_error:307: Connect error: conn_id=3; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 05:15:56,947 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 05:16:04,927 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 05:16:04,932 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 05:16:04,939 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 05:16:04,958 [open_context_base.py] on_connected:289: Connected : conn_id=4; 
2018-11-14 05:16:04,950 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 05:16:04,980 [open_context_base.py] on_connected:289: Connected : conn_id=5; 
2018-11-14 05:16:04,996 [open_context_base.py] on_connected:289: Connected : conn_id=6; 
2018-11-14 05:16:05,006 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=5; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 16230756613053002500, 'conn_key': '08B3D834253C3FD6', 'keep_alive_interval': 10};
2018-11-14 05:16:05,008 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=4; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 2972157111050974727, 'conn_key': '0036D834253A3FBE', 'keep_alive_interval': 10};
2018-11-14 05:16:05,032 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=6; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 5926522864653075204, 'conn_key': '0583E034253E3F49', 'keep_alive_interval': 10};
2018-11-14 05:16:12,998 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 05:16:13,001 [network_manager.py] do_send:295: Send fail: proto_id=2001; serial_no=57078; conn_id=7; msg=Not connected;
2018-11-14 05:16:13,003 [open_context_base.py] on_connected:289: Connected : conn_id=7; 
2018-11-14 05:16:13,006 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=7; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 17383682515835064583, 'conn_key': '0191843C25403FE9', 'keep_alive_interval': 10};
2018-11-14 08:22:14,422 [open_context_base.py] on_error:307: Connect error: conn_id=5; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 08:22:14,424 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 08:22:14,427 [open_context_base.py] on_error:307: Connect error: conn_id=5; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 08:22:14,430 [open_context_base.py] on_error:307: Connect error: conn_id=6; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 08:22:14,430 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 08:22:14,437 [open_context_base.py] on_error:307: Connect error: conn_id=7; err=[WinError 10054] An existing connection was forcibly closed by the remote host;
2018-11-14 08:22:14,438 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 08:22:22,437 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 08:22:22,439 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 08:22:22,440 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 08:22:22,444 [open_context_base.py] on_connected:289: Connected : conn_id=8; 
2018-11-14 08:22:22,446 [open_context_base.py] _wait_reconnect:407: Wait reconnect in 8 seconds: host=127.0.0.1; port=22221;
2018-11-14 08:22:22,449 [network_manager.py] do_send:295: Send fail: proto_id=2001; serial_no=77428; conn_id=9; msg=Not connected;
2018-11-14 08:22:22,455 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=8; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 2961769104031523079, 'conn_key': '00A186D84E521ABE', 'keep_alive_interval': 10};
2018-11-14 08:22:22,456 [open_context_base.py] on_connected:289: Connected : conn_id=9; 
2018-11-14 08:22:22,460 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=9; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 16220368606033646852, 'conn_key': '089588D84E541AD6', 'keep_alive_interval': 10};
2018-11-14 08:22:30,450 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 08:22:30,455 [open_context_base.py] on_connected:289: Connected : conn_id=11; 
2018-11-14 08:22:30,457 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=11; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 17373294508815084935, 'conn_key': '81ED2AE04E581AE9', 'keep_alive_interval': 10};
2018-11-14 23:20:07,123 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 23:20:07,128 [open_context_base.py] on_connected:289: Connected : conn_id=1; 
2018-11-14 23:20:07,132 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=1; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 12905725639681948674, 'conn_key': '06BCE481175A1ADB', 'keep_alive_interval': 10};
2018-11-14 23:20:07,134 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 23:20:07,141 [open_context_base.py] on_connected:289: Connected : conn_id=2; 
2018-11-14 23:20:07,145 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=2; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 9735193701036599044, 'conn_key': '0327E881175C1A3E', 'keep_alive_interval': 10};
2018-11-14 23:20:07,159 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 23:20:07,161 [open_context_base.py] on_connected:289: Connected : conn_id=3; 
2018-11-14 23:20:07,164 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=3; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 2601494090305340930, 'conn_key': '0686ED81175E1A0D', 'keep_alive_interval': 10};
2018-11-14 23:20:42,830 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 23:20:42,832 [open_context_base.py] on_connected:289: Connected : conn_id=1; 
2018-11-14 23:20:42,836 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=1; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 439768468761510405, 'conn_key': '0682F2A317601A47', 'keep_alive_interval': 10};
2018-11-14 23:20:42,841 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 23:20:42,845 [open_context_base.py] on_connected:289: Connected : conn_id=2; 
2018-11-14 23:20:42,848 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=2; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 12905734436346442888, 'conn_key': '82B4F4A317621A4D', 'keep_alive_interval': 10};
2018-11-14 23:20:42,858 [open_context_base.py] _socket_reconnect_and_wait_ready:207: Start connecting: host=127.0.0.1; port=22221;
2018-11-14 23:20:42,860 [open_context_base.py] on_connected:289: Connected : conn_id=3; 
2018-11-14 23:20:42,864 [open_context_base.py] _handle_init_connect:378: InitConnect ok: conn_id=3; info={'server_version': 105, 'login_user_id': 7646405, 'conn_id': 4835286103122025609, 'conn_key': '849CF8A317641A8B', 'keep_alive_interval': 10};
